blockscout-ens/bens-proto/proto/bens.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+
+package blockscout.bens.v1;
+
+option go_package = "github.com/blockscout/blockscout-rs/bens";
+
+service DomainsExtractor {
+  rpc SearchAddress(SearchAddressRequest) returns (SearchAddressResponse) {}
+  rpc GetDetailedDomain(GetDetailedDomainRequest) returns (GetDetailedDomainResponse) {}
+}
+
+enum DomainSource {
+  /// https://docs.ens.domains/
+  ENS = 0;
+  // /// https://docs.unstoppabledomains.com/
+  //UNSTOPPABLE_DOMAINS = 1;
+}
+
+message Domain {
+  /// Source of domain name. Now could be only ENS, in future we could add UNSTOPPABLE_DOMAINS
+  DomainSource domain_source = 1;
+  /// Unique id for the domain, also known as nodehash
+  string id = 2;
+  /// The human readable name, if known. Unknown portions replaced with hash in square brackets (eg, foo.[1234].eth)
+  string name = 3;
+  /// Integer representation of labelhash
+  uint64 token_id = 4;
```

Comment:
Could you please add an example of what is that

Reply:
https://etherscan.io/name-lookup-search?id=levvv.eth